app/User.php

```diff
@@ -496,4 +496,77 @@ public function hasPayPal(): bool
         // @phpstan-ignore-next-line
         return $this->subscribed('kanka') && $this->subscriptions()->first() && str_contains($this->subscriptions()->first()->stripe_price, 'paypal');
     }
+
+    public function upgradePrice(string $period, string $tier): string
+    {
+        $plans = [
+            config('subscription.owlbear.eur.yearly'),
+            config('subscription.owlbear.usd.yearly'),
+            config('subscription.wyvern.eur.yearly'),
+            config('subscription.wyvern.usd.yearly'),
+            config('subscription.elemental.eur.yearly'),
+            config('subscription.elemental.usd.yearly'),
+        ];
+
+
+        $oldPrice = '';
+        $currency = "US$ ";
+        $monthly = true;
+        if ($this->billedInEur()) {
+            $currency = "€ ";
+        }
+
+        $price = "55.00";
+        if ($tier === 'Wyvern') {
+            $price = "110.00";
+            if ($period == 'monthly') {
+                $price = "10.00";
+            }
+        } elseif ($tier === 'Elemental') {
+            $price = "275.00";
+            if ($period == 'monthly') {
+                $price = "25.00";
+            }
+        }
+
+        if ($this->isSubscriber()) {
```

Comment:
That's a big if, could flip it around `if (!$this->isSubscriber() { return $price; }`